File: shortest_word.php
<?php
  // https://www.codewars.com/kata/shortest-word

  // Simple, given a string of words, return the length of the shortest word(s).

  // String will never be empty and you do not need to account for different data types.

  function findShort($str){
    $words = (explode(" ", $str));
    $lengths = array_map("strlen", $words);
    return min($lengths);
  }

  findShort("this is a test to find the shortest word"); // 1
?>

File: python/shortest_word.py
# https://www.codewars.com/kata/57cebe1dc6fdc20c57000ac9

# Simple, given a string of words, return the length of the shortest word(s).

# String will never be empty and you do not need to account for different data types.

def find_short(s):
  s = s.split(' ')
  l = len(s[0])
  for x in s:
    print x
    if len(x) < l:
      l = len(x)
  return l